src/itype.h

```diff
@@ -555,6 +555,10 @@ struct itype {
     long invoke( player *p, item *it, point pos, const std::string &iuse_name ) const;
     long tick( player *p, item *it,  point pos ) const;
 
+    long invoke( player *p, item *it, const tripoint &pos ) const; // Picks first method or returns 0
```

Comment:
The 2D versions are of those functions are still declared above, but they are not implemented.